sklearn/preprocessing/data.py

```diff
@@ -1673,7 +1676,7 @@ def add_dummy_feature(X, value=1.0):
         return np.hstack((np.ones((n_samples, 1)) * value, X))
 
 
-def _transform_selected(X, transform, selected="all", copy=True):
+def _transform_selected(X, transform, selected="all", copy=True, order=False):
```

Comment:
Separate issue: now that other classes from modules outside of `data.py` are using this function, maybe we should create a `utils.py` and place `transform_selected` in it.

Might add that as a todo for a future pull request...